src/components/forms/ComboBox/ComboBox.stories.tsx

```diff
@@ -72,7 +74,7 @@ export const withLabel = (): React.ReactElement => {
 
   return (
     <Form onSubmit={noop}>
-      <Label htmlFor="fruit">Select a fruit</Label>
+      <Label htmlFor="fruit">Select a Fruit</Label>
```

Comment:
The labels should be sentence case ([see USWDS example](https://designsystem.digital.gov/components/combo-box/)):

```suggestion
      <Label htmlFor="fruit">Select a fruit</Label>
```

Reply:
(Same for the others added on this PR